devops/scripts/boot-strap-venv.sh

```diff
@@ -48,12 +48,12 @@ function virtualenv_bootstrap() {
 
         if [ ! -d "$VENV" ]
         then
-            p=$(which "python${PYTHON_VERSION}")
-            echo "Creating Python ${PYTHON_VERSION} virtualenv in ${VENV}"
+            p=$(which "python${PYTHON_VERSION}" 2> /dev/null || which python3)
+            echo "Creating ${p} virtualenv in ${VENV}"
             virtualenv -p "${p}" "${VENV}"
         fi
 
-        "${VENV}/bin/pip" install -q -r "securedrop/requirements/python${PYTHON_VERSION}/develop-requirements.txt"
+        "${VENV}/bin/pip" install -q -r "securedrop/requirements/python${PYTHON_VERSION::1}/develop-requirements.txt"
```

Comment:
Cool trick, but I think we can just hardcode the 3 now. It's already hardcoded in other places too